src/core/executeCommandFactory.js

```diff
@@ -43,7 +43,9 @@ export default ({ logger, configureCommand, debugCommand, handleError }) => {
             componentRegistry => {
               const command = componentRegistry.getCommand(commandName);
               if (!isFunction(command)) {
-                throw new Error(`The ${commandName} command does not exist.`);
+                throw new Error(
+                  `The ${commandName} command does not exist. List of available commands: ${componentRegistry.getCommandNames()}.`
```

Comment:
I would recommend doing `componentRegistry.getCommandNames().join(", ")` just so it's proper English.

Reply:
And change the unit test accordingly also.